src/core/xref.js

```diff
@@ -128,9 +130,9 @@ function collectKeys(xrefs) {
 // adds data to cache
 async function cacheResults(data, cache) {
   const promisesToSet = Object.entries(data).map(([term, results]) =>
-    IDB.set(term, results, cache)
+    idbSet(term, results, cache)
```

Comment:
Oh, doesn't this line requires `return`? Otherwise the `promisesToSet` gets nothing.

Reply:
It's using implicit return as it's an arrow function not surrounded by `{ }`

Reply:
Ah right. My bad 😱